src/tests/no-restricted-imports-dynamic.test.ts

```
@@ -0,0 +1,120 @@
+      code: `import React from 'react';`,
+      options: [{ paths: ['lodash'] }],
+    },
+    // Type imports from restricted paths with allowTypeImports
+    {
+      code: `import type { Something } from 'lodash';`,
+      options: [{ paths: [{ name: 'lodash', allowTypeImports: true }] }],
+    },
+    // Dynamic imports from restricted paths with allowDynamicImports
+    {
+      code: `import { Something } from 'lodash.dynamic';`,
+      options: [{ paths: [{ name: 'lodash.dynamic', allowDynamicImports: true }] }],
```

ESLint

[error] Replace `{·paths:·[{·name:·'lodash.dynamic',·allowDynamicImports:·true·}]·}` with `⏎········{·paths:·[{·name:·'lodash.dynamic',·allowDynamicImports:·true·}]·},⏎······`

(prettier/prettier)